### ISSUE TITLE
Slicing-by-8 or slicing-by-4 support

### ISSUE THREAD

**Issue by stanhu** [tpircher-zz/pycrc#3]
pycrc could be even faster if it supported the slicing-by-8 or slicing-by-4 algorithms, which would just require additional tables to be generated:

https://matt.sh/redis-crcspeed
http://create.stephan-brumme.com/crc32/

Is this something you would want pycrc to support?

**Comment by tpircher-zz** [comment]
Yes, I think pycrc should support this. I'm a bit busy right now, but I hope I can have a look into this in the not-so-distant future.

**Comment by tpircher-zz** [comment]
Just an update. An early preview is now in master on GitHub. This is still experimental, undocumented and brittle. It is known to work for crc-32, may break for other models and will definitely break for models such as crc-5, but I'm working on it.
